src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaConversationPersistenceAdapter.kt

```diff
@@ -27,6 +27,7 @@ class JpaConversationPersistenceAdapter(
     val conversationEntity = conversationRepository.findByReference(updateConversationDto.reference)
     return if (conversationEntity != null) {
       conversationEntityMapper.updateEntityFromDto(conversationEntity, updateConversationDto)
+      conversationEntity.updateLastUpdatedAt() // force the main Induction's JPA managed fields to update
```

Comment:
After our conversation @nathanrussell-moj-digital I took a look at Inductions to see if it had a similar problem and found this line was forcing the JPA managed fields to update even when the parent entity itself had no updates. This seemed like the easiest solution to the problem I was having so replicated here.